### PR TITLE
Polish PageEditor

### PR DESCRIPTION
**Pull request by vanwagonet**
use ItemPicker for who can edit
fix scrollbar weirdness in rce

refs: MBL-14652
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/2qctlqm) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F2qctlqm)
teacher: [Link](https://rebrand.ly/6jkxuwu) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F6jkxuwu)
parent: [Link](https://rebrand.ly/un35igy) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fun35igy)

**Review by inst-danger — COMMENTED** [review]
fix lint

File: Core/Core/Pages/PageEditor/PageEditorView.swift [hunk 1]
```diff
@@ -138,10 +138,21 @@ public struct PageEditorView: View {
...
                 }
-                Button(action: { self.editingRolesPickerShown.toggle() }, label: {
+                Button(action: {
+                    guard let controller = self.viewController() else { return }
+                    let options: [RoleOption] = self.context.contextType == .group
+                        ? [ .members, .public ]
+                        : [ .teachers, .teachersAndStudents, .public ]
+                    self.env.router.show(ItemPickerViewController.create(
+                        title: NSLocalizedString("Can Edit", bundle: .core, comment: ""),
+                        sections: [ItemPickerSection(items: options.map { ItemPickerItem(title: $0.string) } )],
```
Comment (inst-danger) [inline Core/Core/Pages/PageEditor/PageEditorView.swift:148]:
```suggestion
                        sections: [ItemPickerSection(items: options.map { ItemPickerItem(title: $0.string) })],
```

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


<details><summary>:x: Swift lint</summary>


    ⚠️  /Users/vagrant/git/Core/Core/Pages/PageEditor/PageEditorView.swift:148:108: Closing Brace Spacing Violation: Closing brace with closing parenthesis should not have any whitespaces in the middle. (closing_brace)
    [student]
    [teacher]
</details>


[MBL-14652](https://instructure.atlassian.net/browse/MBL-14652)

#### Affected Apps: Student, Teacher
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by vanwagonet** [comment]
You can dismiss the keyboard "interactively" so a swipe down should get rid of it.

**Pull request merged**